src/core/server/base.js

```diff
@@ -1,10 +1,9 @@
-/* eslint-disable no-console */
```

Comment:
If we use bunyan or similar for server logs then this should go away. Plus we have the option of leaving logs in place as long as they're configurable via log levels.